src/components/graph-fieldset/index.tsx

```diff
@@ -173,7 +173,7 @@ const GraphFieldset: FC<Props> = ({
                 uploaded={!!inputGraphFile && !isDragActive}
                 {...getRootProps({ isDragActive, disabled: isLoading })}
               >
-                <input {...getInputProps()} />
+                <input {...getInputProps()} aria-label='input for fil' />
```

Comment:
Den bør støtte flere språk

Reply:
En fordel med å bruke `aria-labelledby` er at den kan peke til tekst som endres basert på språkvalg.

Reply:
Slik det er nå støtter vell siden kun norsk bokmål, så å legge inn støtte for flere språk burde kanskje legges inn som et eget issue? @jeffreiffers

Reply:
Ja det høres riktig ut @hegeaal .